erigon-lib/diagnostics/client.go

```diff
@@ -36,6 +36,10 @@ type DiagnosticClient struct {
 	networkSpeedMutex   sync.Mutex
 }
 
+func NewTestDiagnosticClient() (*DiagnosticClient, error) {
```

Comment:
usually test related stuff should be put into _test.go file, eg this function is only test related - better to move to tests

Reply:
done